docs/reference/object-detection-2d-nms-seq2seq_nms.md

```diff
@@ -0,0 +1,276 @@
+  If *'pretrained'*, downloads a pretrained detector model.
+  If *'images'*, downloads an image to perform inference on. If
+  *'test_data'* downloads a dummy dataset for testing purposes.
+- **verbose**: *bool default=True*\
+  If True, enables maximum verbosity.
+- **url**: *str, default=OpenDR FTP URL*\
+  URL of the FTP server.
+  
+#### Examples
+
+* **Training example.**
+  To train seq2seq-nms properly, the PETS and COCO datasets are supported as Dataset_NMS types. 
+
+  ```python
+  from opendr.perception.object_detection_2d.nms import Seq2SeqNMSLearner
+  import os
+  OPENDR_HOME = os.environ['OPENDR_HOME']
+  
+  seq2SeqNMSLearner = Seq2SeqNMSLearner(fmod_map_type='EDGEMAP', iou_filtering=0.8, 
+                                        app_feats='fmod', checkpoint_after_iter=1,
+                                        temp_path=OPENDR_HOME + '/src/opendr/perception/'
+                                                              'object_detection_2d/nms/seq2seq_nms/temp',
+                                        epochs=8)
+  seq2SeqNMSLearner.fit(dataset='PETS', use_ssd=False,
+                        datasets_folder=OPENDR_HOME + '/src/opendr/perception/object_detection_2d/nms/datasets',
+                        logging_path='./logs_pets_exp1', silent=False, verbose=True, nms_gt_iou=0.50,
+                        max_dt_boxes=500)
+  ```
+
+* **Inference and result drawing example on a test .jpg image using OpenCV.**
+
+  ```python
+  from opendr.perception.object_detection_2d.nms import Seq2SeqNMSLearner
+  from opendr.engine.data import Image
+  from opendr.perception.object_detection_2d import SingleShotDetectorLearner
+  from opendr.perception.object_detection_2d import draw_bounding_boxes
+  import os
+  OPENDR_HOME = os.environ['OPENDR_HOME']
+
+  seq2SeqNMSLearner = Seq2SeqNMSLearner(fmod_map_type='EDGEMAP', iou_filtering = 0.8,
+                                        app_feats='fmod', device='cpu')
+  seq2SeqNMSLearner.load(OPENDR_HOME + '/src/opendr/perception/object_detection_2d/nms/seq2seq_nms/pets_exp0/'
+                                       'checkpoint_epoch_7', verbose=True)
+  ssd = SingleShotDetectorLearner(device='cuda')
+  ssd.download(".", mode="pretrained")
+  ssd.load("./ssd_default_person", verbose=True)
+  img = Image.open(OPENDR_HOME + '/projects/perception/object_detection_2d/nms/img_temp/frame_0000.jpg')
+  if not isinstance(img, Image):
+      img = Image(img)
+  boxes = ssd.infer(img, threshold=0.25, custom_nms=seq2SeqNMSLearner)
+  draw_bounding_boxes(img.opencv(), boxes, class_names=ssd.classes, show=True)
+  ```
+  
+#### Performance Evaluation
+
+TABLE-1: Average Precision (AP) achieved by pretrained models on the person detection task. The maximum number or RoIs, employed for the performance evaluation was set to 800.
+![Alt text](https://raw.githubusercontent.com/opendr-eu/opendr/nms-module/src/opendr/perception/object_detection_2d/nms/seq2seq_nms/stats_pretrained.png?raw=true "Title")
```

Comment:
I feel bad having to say this since it's a nice looking table, but since everybody else added theirs using a markdown table for consistency we should do it here as well. If changes need to be done, it's easier to do them.

You can use a tool like this to simplify the process: https://www.tablesgenerator.com/markdown_tables

Reply:
Done